Bottom: 7432f96719d384cc099d7893e696dbac9f06a634
Top:    eba77f5b63e65d9b4e30b612e62e0233c1e8209c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-23 19:42:18 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!






---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 2dcf4b8..5e56536 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -312,13 +312,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
